components/console/helpers/progressbar.rst

```diff
@@ -289,17 +289,23 @@ to display it can be customized::
 
 .. caution::
 
-    For performance reasons, be careful if you set the total number of steps
-    to a high number. For example, if you're iterating over a large number of
-    items, consider setting the redraw frequency to a higher value by calling
-    :method:`Symfony\\Component\\Console\\Helper\\ProgressBar::setRedrawFrequency`,
+    By default, for performance reasons Symfony redraws screen every 100ms.
+    In special cases, this can be too often and might hurt performance.
+    In other cases, this can be opposite and it hurts UX.
+    In either of these cases, consider tweaking redraw frequency by either of these methods:
+    :method:`Symfony\\Component\\Console\\Helper\\ProgressBar::preventRedrawFasterThan`
+    :method:`Symfony\\Component\\Console\\Helper\\ProgressBar::setRedrawFrequency`
+    :method:`Symfony\\Component\\Console\\Helper\\ProgressBar::forceRedrawSlowerThan`
     so it updates on only some iterations::
 
         $progressBar = new ProgressBar($output, 50000);
         $progressBar->start();
 
-        // update every 100 iterations
-        $progressBar->setRedrawFrequency(100);
+        // redraw every 100 iterations or every 200ms (in case iterations go too slow)
```

Comment:
the comment says "100 iterations" but the code says "200", right?

Reply:
good catch! fixed

Reply:
@javiereguiluz I can see this issue is there back, despite it being correct on local branch (setRedrawFrequency(100)). Any idea what happened here?

Reply:
Don't know ... but merged docs use `100` both in the code and the comment: https://github.com/symfony/symfony-docs/blob/4.4/components/console/helpers/progressbar.rst#bar-settings